mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -301,6 +301,21 @@ impl TakerSavedSwap {
             self.taker_coin_usd_price = Some(rates.rel);
         }
     }
+
+    pub fn swap_pubkeys(&self) -> SwapPubkeys {
+        let mut swap_pubkeys = SwapPubkeys::default();
+
+        // TODO: Adjust for private coins when/if they are braodcasted
+        for data in &self.events {
+            if let TakerSwapEvent::Started(started) = &data.event {
+                swap_pubkeys.maker = started.maker_coin_htlc_pubkey;
+                swap_pubkeys.taker = Some(started.my_persistent_pub);
```

Comment:
Same for those 2 pubkeys, aren't they both the taker's?

Reply:
@shamardy this is ready for another round of review!